### ISSUE TITLE
Update PRA wiki page to provide general guidance for anyone creating digital forms in the Federal govt

### ISSUE THREAD

**Issue by jcmeloni-usds** [usds/us-forms-system#103]
This is an amendment of an original issue (#78) created early in the project.

When creating online forms in the Federal govt, you must keep PRA in mind.  PRA may or may not apply to your project.  PRA will apply if you are materially changing the content of the form, such that you are collecting different information than already approved in the offline form (typically when you are creating *more* information).  

The process of creating a *better* experience for users of online vs offline forms -- one of the purposes of this library -- may mean some questions are changed, removed, or even added, and therefore you must have a minmial working knowledge of PRA.  

We should offer that to users of this library.

We have an existing wiki page called "PRA Implications" (https://github.com/usds/us-forms-system/wiki/PRA-Implications). I recommend changing the name of this page to something a bit more friendly, and also editing the content to ensure we are addressing the following:

* what is PRA and why do you need to know about it when creating online versions of offline forms
* where to go for specific questions regarding PRA in geneal
* where to go for super specific questions about your own form(s)
* examples of when PRA does and does not apply when creating online versions of offline forms

**Comment by jcmeloni-usds** [comment]
Assigning initially to @ju-liem and @fatimanoorva to:
* get this on the board
* prioritize
* assign as appropriate (fatima, I think you said this was something you are interested in, so I'd be cool w/ you running with this if j-mac is)

**Comment by annekainicUSDS** [comment]
Information that the Vets.gov team has collected over the years on PRA:

1. https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Work%20Practices/Research/Request%20or%20Do%20Research/PRA%20and%20Recruiting/PRA
2. https://github.com/usds/usds/blob/master/how-to/PRA.md

**Comment by jcmeloni-usds** [comment]
@annekainicUSDS vets.gov-team is private still, right?

**Comment by annekainicUSDS** [comment]
Ah shoot, that's right. I can copy the content in the vets.gov-team link and send to you all via email.

**Comment by jcmeloni-usds** [comment]
@fatima-gov is this one of the things you said you're actively working on?

**Comment by jcmeloni-usds** [comment]
I'd like to keep this here as something for Fatima to sink her teeth into.

**Comment by fatima-gov** [comment]
Closing this for now. Opening a separate ticket to document latest process in parallel with PRA.gov team.

**Comment by annekainicUSDS** [comment]
@fatima-gov once you create that new ticket, do you mind linking to it here? That way we can track it in case someone lands on this ticket. Thanks!
